meerkat_analysis/indicators.py

```diff
@@ -41,13 +41,23 @@ def fix_dates(start_date, end_date, epi_week_start_day):
     return start_date, end_date, freq
 
 
-def count(data, var_id, start_date=None, end_date=None, epi_week_start_day=None):
+def calculate(data, var_id,calc,start_date=None, end_date=None, epi_week_start_day=None):
```

Comment:
PEP8: just missing a space after `var_id,` and after `calc,`

Reply:
Yes, it was the starting point  to understand the problem, I will make it neat and add switch statement etc.